include/c/motes.h

```diff
@@ -96,7 +96,9 @@
 #   define c3__bnpd   c3_s4('b','n','p','d')
 #   define c3__bnps   c3_s4('b','n','p','s')
 #   define c3__bnsp   c3_s4('b','n','s','p')
+#   define c3__boat   c3_s4('b','o','a','t')
 #   define c3__boce   c3_s4('b','o','c','e')
+#   define c3__bog    c3_s3('b','o', 'g')  /* set background color in term */
```

Comment:
are the stray spaces on commented mote lines intentional?
relatedly, trying to document motes here seems very at odds with the style of the rest of the file: I think `motes.h` is meant to be a very mechanical "fully macro-expanded" minor C language extension, while semantics commentary should go in the relevant `term.c` case statements and hoon structure definitions

Reply:
fixed